bindings/python/src/session.cpp

```diff
@@ -57,61 +57,61 @@ namespace lt = libtorrent;
 namespace
 {
 #ifndef TORRENT_NO_DEPRECATE
-    void listen_on(lt::session& s, int min_, int max_, char const* interface, int flags)
```

Comment:
the whitespace changes should be a separate review/PR. I think it would be best to fix *all* python binding source files to conform to the code conventions, not just session.cpp